cobra/core/reaction.py

```diff
@@ -1087,29 +1087,60 @@ def build_reaction_from_string(self, reaction_str, verbose=True,
                     met = Metabolite(met_id)
                 self.add_metabolites({met: num})
 
-    def summary(self, names=False):
+    def summary(
```

Comment:
Do you intend to add some functionality which might require the other parameters to be passed?

Reply:
Not really. It's mostly for interface consistency.